client/src/App.js

```diff
@@ -34,11 +35,13 @@ class App extends Component {
 
   componentDidMount = async () => {
     const hotLoaderDisabled = zeppelinSolidityHotLoaderOptions.disabled;
+    let GaslessCounter = {};
     let Counter = {};
     let Wallet = {};
     try {
-      Counter = require("../../contracts/Counter.sol");
-      Wallet = require("../../contracts/Wallet.sol");
+      GaslessCounter = require("./contracts/GaslessCounter.json");
```

Comment:
We switched to importing `.sol` files instead of `.json` because of [Solidity Hot Loader](https://github.com/ylv-io/zeppelin-solidity-hot-loader). What is the reason to go back to `.json`? Does it works for you?

Reply:
accidentally left .json in from a previous merge. Changed to .sol files.

Reply:
Thanks!